### PR TITLE
feature/AB#197 Improved styling

### PR DESCRIPTION
**Pull request by DerCed**
Improved styling

## Description
Improved Styling of the Accordions and tabs

## Motivation and Context
Looked a little bit too dense. Added spacing between elements

## How has this been tested?
Local Env

## Screenshots (if appropriate):
image
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: webclient/app/src/app/features/apps/entities/entityDialog/EntityDialog.jsx [hunk 1]
```diff
@@ -321,31 +283,35 @@ function EntityDialog(props) {
                     regex={RegexConfig.entityTitle}
                 />
                 <Box className={entityEditorStyles.tabBox}>
-                    <Tabs value={value} onChange={handleTabChange} aria-label="basic tabs example">
+                    <Tabs value={value} onChange={handleTabChange} aria-label="basic tabs example" className={entityEditorStyles.tabHeader}>
```
Comment (DerCed) [inline webclient/app/src/app/features/apps/entities/entityDialog/EntityDialog.jsx:286]: Wichtige Stelle

File: webclient/app/src/app/features/apps/entities/entityDialog/EntityDialog.jsx [hunk 2]
```diff
@@ -321,31 +283,35 @@ function EntityDialog(props) {
...
                         <Tab label={t("entity.fields")} {...a11yProps(0)} />
                         <Tab label={t("entity.relations")} {...a11yProps(1)} />
                     </Tabs>
                     <TabPanel value={value} index={0}>
-                        {renderFields()}
-                        <Button fullWidth startIcon={<Add/>} onClick={addField}>{t("button.create")}</Button>
+                        <Stack spacing={1}>
```
Comment (DerCed) [inline webclient/app/src/app/features/apps/entities/entityDialog/EntityDialog.jsx:291]: Wichtige Stelle

File: webclient/app/src/app/features/apps/entities/entityDialog/EntityDialog.jsx [hunk 3]
```diff
@@ -321,31 +283,35 @@ function EntityDialog(props) {
...
+                            {renderFields()}
+                            <Button fullWidth startIcon={<Add />} onClick={addField}>
+                                {t("button.create")}
+                            </Button>
+                        </Stack>
                     </TabPanel>
                     <TabPanel value={value} index={1}>
-                        {renderRelations()}
-                        <Button fullWidth startIcon={<Add/>} onClick={addRelationship}>{t("button.create")}</Button>
+                        <Stack spacing={1}>
```
Comment (DerCed) [inline webclient/app/src/app/features/apps/entities/entityDialog/EntityDialog.jsx:299]: Wichtige Stelle

File: webclient/app/src/app/features/apps/entities/entityDialog/EntityDialogStyles.js [hunk 4]
```diff
@@ -1,20 +1,23 @@
...
-        height: "15rem"
+        height: "15rem",
     },
     flex: {
-        flex: 1
-    }
-}))
+        flex: 1,
+    },
+    tabHeader: {
```
Comment (DerCed) [inline webclient/app/src/app/features/apps/entities/entityDialog/EntityDialogStyles.js:19]: Wichtige Stelle

File: webclient/app/src/app/features/apps/entities/entityDialog/EntityDialogStyles.js [hunk 5]
```diff
@@ -1,20 +1,23 @@
...
+        paddingBottom: "1rem",
```
Comment (DerCed) [inline webclient/app/src/app/features/apps/entities/entityDialog/EntityDialogStyles.js:20]:
```suggestion
        paddingBottom: theme.spacing(2),
```

File: webclient/app/src/app/features/apps/entities/fieldAccordion/FieldAccordion.jsx [hunk 6]
```diff
@@ -69,61 +39,36 @@ function FieldAccordion(props) {
...
-
-        editFieldProperty("fieldType", event.target.value)
-        editFieldProperty("fieldValidateRulesPattern", "")
-
+        editFieldProperty("fieldType", event.target.value);
+        editFieldProperty("fieldValidateRulesPattern", "");
     }
 
     return (
-        <Grid container spacing={2} alignItems={"center"} justifyItems={"center"}>
```
Comment (DerCed) [inline webclient/app/src/app/features/apps/entities/fieldAccordion/FieldAccordion.jsx:116]: Wichtige Stelle

File: webclient/app/src/app/features/apps/entities/fieldAccordion/FieldAccordion.jsx [hunk 7]
```diff
@@ -69,61 +39,36 @@ function FieldAccordion(props) {
...
+        <Grid container alignItems={"center"} justifyItems={"center"}>
             <Grid item sm={11}>
-                <Accordion>
-                    <AccordionSummary
-                        expandIcon={<ExpandMore/>}
-                        aria-controls="panel1a-content"
-                        id="panel1a-header"
-                    >
+                <Accordion elevation={2}>
```
Comment (DerCed) [inline webclient/app/src/app/features/apps/entities/fieldAccordion/FieldAccordion.jsx:68]: Wichtige Stelle

File: webclient/app/src/app/features/apps/entities/relationshipAccordion/RelationshipAccordion.jsx [hunk 8]
```diff
@@ -41,34 +22,29 @@ function RelationshipAccordion(props) {
     }
 
     return (
-        <Grid container spacing={2} alignItems={"center"} justifyItems={"center"}>
```
Comment (DerCed) [inline webclient/app/src/app/features/apps/entities/relationshipAccordion/RelationshipAccordion.jsx:44]: Wichtige Stelle

**Review by DerCed — COMMENTED** [review]
Sorry, the IDE formatted again....
I've marked the important parts.

File: webclient/app/src/app/features/apps/entities/entityDialog/EntityDialog.jsx (hunk 1 above)
Comment (witchpou) [reply webclient/app/src/app/features/apps/entities/entityDialog/EntityDialog.jsx:286]: :-D

**Review by witchpou — APPROVED** [review]

**Pull request merged**